cmd/eksctl-anywhere/cmd/flags.go

```diff
@@ -0,0 +1,39 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+)
+
+func addClusterFlags(cc *cobra.Command, clusterOpt clusterOptions, override ...string) {
```

Comment:
can we add `--force-cleanup` to this? it looks like all 3 commands use this flag

Reply:
I wanted to group the addClusterFlags together based on fields defined by clusterOptions. I think functionally it would help to add this since all commands use it though. Would it make sense to rename the function and group them all together?

Reply:
Updated the func to add --force-cleanup and rebased to resolve a merge conflict.